build/windows/10.1.0/sitecore-roles/sitecore-custom-base/xm-build.json

```diff
@@ -0,0 +1,107 @@
+{
+  "tags": [
+    {
+      "tag": "community/sitecore-xm1-custom-spe-cm:10.1.0-${windowsservercore_version}",
+      "build-options": [
+        "--build-arg BASE_IMAGE=${sitecore_registry}/sxp/sitecore-xm1-cm:10.1.0-${windowsservercore_version}",
+        "--build-arg MODULE_ASSETS=${sitecore_registry}/sxp/modules/spe-assets:6.2-1809",
+        "--memory 4GB",
+        "--build-arg ROLE=cm",
+        "--target base"
+      ]
+    },
+    {
+      "tag": "community/sitecore-xm1-custom-sxa-cm:10.1.0-${windowsservercore_version}",
+      "build-options": [
+        "--build-arg BASE_IMAGE=community/sitecore-xm1-custom-spe-cm:10.1.0-${windowsservercore_version}",
+        "--build-arg MODULE_ASSETS=${sitecore_registry}/sxp/modules/sxa-xm1-assets:10.1.0-1809",
+        "--memory 4GB",
+        "--build-arg ROLE=cm",
+        "--target content"
+      ]
+    },
+    {
+      "tag": "community/sitecore-xm1-custom-sxa-jss-cm:10.1.0-${windowsservercore_version}",
```

Comment:
Maybe the JSS tagged containers should now be renamed to `Headless` or something ?